### ISSUE TITLE
Combo boxes in issue reporter don't work as expected in NVDA

### ISSUE THREAD

**Issue by Neurrone** [Microsoft/vscode#60090]
Issue Type: <b>Bug</b>

1. Choose help -> report issue.
2. Press the down arrow key to start reading from the top. NVDA should be in browse mode, so I expect to be able to arrow through the entire document.

However, when encountering the issue type combo box, pressing the down arrow keys changes the selection without me going into focus mode. Additionally, you can't use the down arrow key to move past it, so it it seems to be responding to keyboard events it shouldn't be.

VS Code version: Code - Insiders 1.28.0-insider (431ef9da3cf88a7e164f9d33bf62695e07c6c2a9, 2018-10-03T12:44:29.343Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.86GB (10.04GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|40%|

</details><details><summary>Extensions (6)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-markdownlint|Dav|0.20.0
vscode-eslint|dba|1.6.0
python|ms-|2018.8.0
cpptools|ms-|0.19.0
rust|rus|0.4.10
markdown-preview-enhanced|shd|0.3.8

</details>

**Comment by cleidigh** [comment]
@RMacfarlane 
FYI, I don't think you use the select box widget, but I think we may have similar issues navigating the new settings editor. let me know if there's something here we should move into selectbox also.

**Comment by cleidigh** [comment]
@Neurrone 
@RMacfarlane 
As I stated above, I have been looking at this relative to settings and selectbox. I found a pretty relevant and detailed reader mode explanation.  My interpretation is that in NVDA automatically switches to focus mode on a combo box.  Result: DownArrow change of selection once the select box has focus.

I think this may be an "as designed" ?
Additionally NVDA is pretty configurable, perhaps the user can change some of this behavior?

https://tink.uk/understanding-screen-reader-interaction-modes/

**Comment by Neurrone** [comment]
@cleidigh it isn't activating focus mode, in fact I can't turn focus mode off because it is in browse mode. However, it is just stuck there within the 3 options of the combo box.

**Comment by cleidigh** [comment]
@Neurrone 
From playing around, I think the following is happening:
- When issue reporter windows opened , no element has focus , reader should be a browse mode
- First DownArrow skips to second element (combobox) and automatically goes into focus mode on that element
- To move to the next element without changing selection, press Escape then Tab

I do think there may be some tab ordering issues confusing things.

@RMacfarlane 
is it possible there is a missing or wrong order TabIndex?

**Comment by RMacfarlane** [comment]
@cleidigh Thanks for your investigation! The only change I've made to tabIndex is to add `tabIndex=0` to the text `disabling all extensions and reloading the window`. All other elements that receive focus are selects, inputs, or text areas, so the tab order should just be sequential. Are you seeing an unexpected tab order?

I'll add some describedby labeling.

If the user has the display language configured to something that isn't English, a div with text that says `Please complete the form in English.` is revealed, so I think that should be read before the first select.

I'm wondering if some of the event handlers on the issue reporter are getting in the way of NVDA, I'll test that

**Comment by RMacfarlane** [comment]
@Neurrone I'm on the same windows version with version 1.28 of VSCode and haven't been able to reproduce this. When pressing the down arrow, information about the combobox is read, but focus is unchanged and I can still press down to get to the next line. I have version 2018.2.1 of NVDA, what version are you using?

**Comment by Neurrone** [comment]
I can reproduce this with NVDA alpha-16149,e713f614, with both 1.28 stable and the latest insiders. I'll check tomorrow on another computer with the latest NVDA stable.

**Comment by Neurrone** [comment]
It looks like this is a problem introduced by a recent NVDA update, since it works as expected on stable.

**Comment by Neurrone** [comment]
Btw, is there an easy way for me to get the html markup used in the issue reporter, and the associated javascript for me to extract it out into a normal webpage for easier inspection?
